addon/components/polaris-text-field/resizer.js

```diff
@@ -0,0 +1,132 @@
+import Component from '@ember/component';
+import layout from '../../templates/components/polaris-text-field/resizer';
+import { htmlSafe } from '@ember/string';
+import { computed } from '@ember/object';
+import { scheduleOnce } from '@ember/runloop';
+
+const REPLACE_REGEX = /[\n&<>]/g;
+
+const ENTITIES_TO_REPLACE = {
+  '&': '&amp;',
+  '<': '&lt;',
+  '>': '&gt;',
+  '\n': '<br>',
+};
+
+const replaceEntity = (entity) => {
+  return ENTITIES_TO_REPLACE[entity] || entity;
+};
+
+export default Component.extend({
+  classNames: ['Polaris-TextField__Resizer'],
+
+  'aria-hidden': true,
+
+  layout,
+
+  /**
+   * The value of the textarea
+   *
+   * @property contents
+   * @public
+   * @type {String}
+   * @default null
+   */
+  contents: null,
+
+  /**
+   * The height (in px) of the textarea
+   *
+   * @property currentHeight
+   * @public
+   * @type {Number}
+   * @default null
+   */
+  currentHeight: null,
+
+  /**
+   * The multiline value of the textarea if
+   * a numeric value was passed-in to the polaris-text-field
+   *
+   * @property minimumLines
+   * @public
+   * @type {Number}
+   * @default null
+   */
+  minimumLines: null,
+
+  /**
+   * Callback when the height of the resize container changes
+   *
+   * @property onHeightChange
+   * @public
+   * @type {Function}
+   * @default noop
+   */
+  onHeightChange(/* height */) {},
+
+  finalContents: computed('contents', function() {
+    let contents = this.get('contents');
+
+    contents = contents
```

Comment:
The linter doesn't complain about having the `?` and `:` on the next line? 😮

Reply:
Nope! But I feel like the Andy linter does, so if you'd like I can shift things onto one line, although we risk having it look different than the React code and harder to pick out differences.

Reply:
Nah I don't mind, just surprised is all 😛